### PR TITLE
Updated RDS DB Version in variables.tf

### PR DESCRIPTION
**Pull request by techwithshadab**
**Which issue is resolved by this Pull Request:**
Current RDS DB instance version is not available, and giving the below error
```
Error: creating RDS DB Instance (terraform-20230228011128580900000008): InvalidParameterCombination: Cannot find version 8.0.30 for mysql
│ 	status code: 400, request id: f07842c2-e27e-437e-833c-55cf60ebb309
│ 
│   with module.kubeflow_components.module.rds[0].aws_db_instance.kubeflow_db,
│   on ../../../iaac/terraform/aws-infra/rds/main.tf line 41, in resource "aws_db_instance" "kubeflow_db":
│   41: resource "aws_db_instance" "kubeflow_db" {

Makefile:30: recipe for target 'deploy-kubeflow-components' failed
```
**Description of your changes:**
Updated version `8.0.30` to `8.0.32`

**Testing:**
- Tested and its working fine after version update

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: iaac/terraform/aws-infra/rds/variables.tf [hunk 1]
```diff
@@ -45,7 +45,7 @@ variable "db_allocated_storage" {
 variable "mysql_engine_version" {
   type        = string
   description = "The engine version of MySQL"
-  default = "8.0.30"
+  default = "8.0.31"
```
Comment (ananth102) [inline iaac/terraform/aws-infra/rds/variables.tf:48]: Thanks for this PR! Can you use 8.0.32

Comment (techwithshadab) [reply iaac/terraform/aws-infra/rds/variables.tf:48]: Yes, tested for that too. It's working fine

Comment (techwithshadab) [reply iaac/terraform/aws-infra/rds/variables.tf]: I have updated the MySQL DB instance version for Cognito-RDS-S3 and RDS-S3 deployment.

**Review by ananth102 — APPROVED** [review]
Thanks for the PR! All relevant terraform integration tests(rds-s3/cognito-rds-s3) have passed.

**Pull request merged**